packages/elements/src/components/ino-label/ino-label.scss

```diff
@@ -23,14 +23,6 @@ ino-label {
     }
   }
 
-  &:not([ino-outline]) .mdc-floating-label {
```

Comment:
These changes break the filled `ino-select` styles. If there are specific styles that should be applied to the label of the filled textarea, we should overwrite them in the textarea stylesheet.

Reply:
![image](https://user-images.githubusercontent.com/22963121/103524030-2b580780-4e7d-11eb-867c-65d7a841a127.png)

Reply:
Ok, I adjusted this as you suggested.